cmd/eksctl-anywhere/cmd/importimages.go

```diff
@@ -52,11 +51,6 @@ var importImagesCmd = &cobra.Command{
 }
 
 func importImages(ctx context.Context, spec string) error {
-	registryUsername := os.Getenv("REGISTRY_USERNAME")
-	registryPassword := os.Getenv("REGISTRY_PASSWORD")
-	if registryUsername == "" || registryPassword == "" {
-		return fmt.Errorf("username or password not set. Provide REGISTRY_USERNAME and REGISTRY_PASSWORD for importing helm charts (e.g. cilium)")
-	}
```

Comment:
If you do need auth, should probably use `HELM_REGISTRY_CONFIG=~/.docker/config.json` to keep things simple

Reply:
That's a great suggestion! We actually don't need the auth however